handlers/handler-loadbalance/src/main/java/org/apache/servicecomb/loadbalance/LoadbalanceHandler.java

```diff
@@ -98,6 +98,9 @@ public Server chooseServer(Object key) {
       boolean isRetry = null != lastServer;
       for (int i = 0; i < COUNT; i++) {
         Server s = delegate.chooseServer((Invocation) key);
+        if (s == null) {
+          break;
+        }
```

Comment:
unneccessary null check

Reply:
Already updated.
break the loop when s == null
only continue the loop and retry another server when s.equals(lastServer)